pkgs/jnigen/lib/src/elements/elements.dart

```diff
@@ -1120,7 +1123,9 @@ class KotlinType implements Element<KotlinType> {
   final String kind;
   final String? name;
   final int id;
-  final List<KotlinTypeProjection> arguments;
+
+  /// `null` represents a wildcard.
```

Comment:
Would it make sense to add an AST element for wildcards, rather than using null to represent it? If wildcards are only permitted in this arguments field, you could add a `KotlinTypeArgument` class that can be either a wildcard or a `KotlinTypeProjection`. That would be more self documenting.

Reply:
I get this from a JSON and use `json_serializable` to autogenerate the deserialization. I could do a custom thing for this one class though.